specs/npm/find_and_delete_node_modules_in_a_folder.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Find and delete node_modules in a folder
+command: find . -name 'node_modules' -type d -prune -print -exec rm -rf '{}' \;
+tags:
+  - npm
+description: Finds and deletes all node_modules in a folder
+arguments: []
+source_url: "https://stackoverflow.com/questions/22675725/find-unused-npm-packages-in-package-json"
```

Comment:
Is this source url correct for these workflows? I don't see the corresponding commands mentioned within the stackoverflow post